pkg/web/apps/apps_http_handler.go

```diff
@@ -13,7 +14,8 @@ type (
 	HTTPHandler interface {
 		GetApps(c echo.Context) error
 		GetAppByID(c echo.Context) error
-		UpdateApp(c echo.Context) error
+		UpdateAppVersions(c echo.Context) error
```

Comment:
@davidffrench is this implementation in the way that you are looking for? I mean, can we move with the UpdateAppVersions and UpdateAllAppVersions and the endpoints mapped in this way?